gst/tensor_decoder/tensordec.c

```diff
@@ -761,11 +762,19 @@ gst_tensordec_label_set_output (GstTensorDec * self, GstBuffer * outbuf,
   g_assert (out_mem != NULL);
   g_assert (gst_memory_map (out_mem, &out_info, GST_MAP_WRITE));
 
+  /**
+   * Security: strncpy protects from buffer overflows. But, if it prevents an
+   * overflow without null termiating, a subsequent string operation cause overflow.
+   *
+   * Gstreamer: Note that gst-plugins-base/gstbasetextoverlay.c:g_utf8_validate()
+   * displays '*' at back of the string if a string includes a NULL character.
+   */
   strncpy ((char *) out_info.data, label, len);
 
+  gst_memory_unmap (out_mem, &out_info);
```

Comment:
Let's append a buffer memory after unmap the existing memory.
(Reviewed by @jaeyun-jung. )